### DISCUSSION TITLE
Apple's 3rd party SDK requirements for Capacitor

### DISCUSSION THREAD

**Discussion by dpa99c** [ionic-team/capacitor#7146] [General]
Apple has listed Capacitor on its list of 3rd party SDKs that will need to provide privacy info in order to be included in iOS apps:
https://developer.apple.com/support/third-party-SDK-requirements/

This is also the case for Cordova (see https://github.com/apache/cordova-ios/issues/1391), so I just wanted to start a discussion about how this might be handled and whether Capacitor will require a similar or different approach than Cordova to handling this.

**Comment by SpenserJ** [comment]
Reposting this without mentioning the Ionic team, since I believe that is why my previous comment was marked as disruptive ☹️ 

Since the Ionic team often struggles with communication (no shame, I'm sure there is a lot going on), I'll try to give a quick summary of where I believe things are at. I'm not on the Ionic team, so take all of this with a grain of salt, but this is what I've observed so far.

In the past few days, three PRs have been merged to add the XCPrivacy manifest into https://github.com/ionic-team/capacitor/pull/7330, https://github.com/ionic-team/capacitor/pull/7322, and https://github.com/ionic-team/capacitor/pull/7321. Last November, the Ionic team announced a new [Capacitor Release Cadence](https://ionic.io/blog/introducing-a-new-capacitor-release-cadence), which boils down to:

August-December: Prerelease of the next major update (6.x)
December-February: Next major update is considered stable, but not tagged as latest, so that plugin developers have some time to update
February-February: That version is tracked as the latest while the previous version drops into maintenance mode for 6 months, and then only has external (3rd party?) support
It looks like the Ionic team is behind schedule on 6.x, as the [first alpha was in November](https://github.com/ionic-team/capacitor/releases/tag/6.0.0-alpha.1), and the [first/only/latest RC was at the end of January](https://github.com/ionic-team/capacitor/releases/tag/6.0.0-rc.0), and they still haven't released 6.0.0 stable (scheduled for last December). Its not clear if we'll see a 6.0.0 stable release arrive before Apple's privacy SDK requirements kick in, and if they stick to the release cadence then it wouldn't be tagged as latest for another 2 months.

Thankfully the changes were also merged into 4.x (maintenance?) and 5.x (current stable/latest), so we'll hopefully see a new 5.x version released within a week or two. Sadly this is cutting it close for a lot of teams, as Apple's [latest announcement](https://developer.apple.com/news/?id=3d8a9yyh) from February 29th said that warnings will start appearing on March 13th (yesterday) and it will be enforced on May 1st (6 weeks from now). I'd greatly appreciate some additional context from the Ionic team on when a new 5.x version will be released, as I have quite a few people breathing down my neck for when we'll have these issues resolved in our apps.

**Reply by giralte-ionic** [reply]
We're finishing up and releasing 4.x, 5.x and 6.0rc2 likely tomorrow pending some final tests.

**Reply by SpenserJ** [reply]
Amazing, thank you so much for the update! Even if it doesn't land tomorrow, its reassuring to hear that an update should be coming out pretty quick 😅
